examples/deadline/All-In-AWS-Infrastructure-SEP/ts/.gitignore

```diff
@@ -0,0 +1,20 @@
+*.d.ts
+*.js
+cdk.out
+cdk.context.json
+
+# Used by nyc
+.nyc_output
+coverage
+.nycrc
+
+# Exclude package artifacts
+dist
+*.tgz
+.LAST_PACKAGE
+
+#Defines license that must be present
+!license-header.js
+
+# The staged files for Deadline
+stage
```

Comment:
nit: File should end with a newline.

There are a couple of files like this. I'll save you the copy/paste by not commenting on all of them.